core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -63,6 +63,7 @@
 // The test suite itself is Serializable so that anonymous Function implementations can be
 // serialized, as an alternative to converting these anonymous classes to static inner classes;
 // see http://stackoverflow.com/questions/758570/.
+@SuppressWarnings("serial")
```

Comment:
There are ton of classes with serializable but missing uid. I'm just going to remove this.